hazelcast/src/main/resources/hazelcast-client-config-5.2.xsd

```diff
@@ -702,47 +703,46 @@
             </xs:element>
         </xs:choice>
     </xs:complexType>
-    <xs:complexType name="registered-classes">
+
+    <xs:complexType name="compact-serializers">
```

Comment:
why do we have `compact-` prefix?

Reply:
See my comment above, this is like this to avoid possible confusion, as the type definition is not under the compact serializer tag